x-pack/plugins/security/index.js

```diff
@@ -114,9 +113,11 @@ export const security = (kibana) => new kibana.Plugin({
     server.auth.strategy('session', 'login', 'required');
 
     const auditLogger = new SecurityAuditLogger(server.config(), new AuditLogger(server, 'security'));
-    const checkPrivilegesWithRequest = checkPrivilegesWithRequestFactory(server);
-    const { savedObjects } = server;
 
+    // exposes server.plugins.security.authorization
+    initAuthorization(server);
```

Comment:
nit: what about calling this something a little more descriptive, like `initAuthorizationService`? This more closely matches what you describe in `init.test.js`